<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cardform</class>
 <widget class="QDialog" name="cardform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New record</string>
  </property>
  <widget class="QGroupBox" name="cardForm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="button_cardSave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save card</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>0000 0000 0000 0000 000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CVC2/CVV2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardCVC">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Owner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardOwner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Secret</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardSecret">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Support Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardPhone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_acount_name_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_cardPin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_secret">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/eye_closed.png</normaloff>
      <normalon>:/images/eye_opened.png</normalon>:/images/eye_closed.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_pin">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>180</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/eye_closed.png</normaloff>
      <normalon>:/images/eye_opened.png</normalon>:/images/eye_closed.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>field_cardName</tabstop>
  <tabstop>field_cardNumber</tabstop>
  <tabstop>field_cardDate</tabstop>
  <tabstop>field_cardCVC</tabstop>
  <tabstop>field_cardSecret</tabstop>
  <tabstop>field_cardOwner</tabstop>
  <tabstop>field_cardPin</tabstop>
  <tabstop>field_cardPhone</tabstop>
  <tabstop>button_cardSave</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
